builtin/providers/aws/resource_aws_api_gateway_base_path_mapping.go

```diff
@@ -0,0 +1,130 @@
+package aws
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayBasePathMapping() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayBasePathMappingCreate,
+		Read:   resourceAwsApiGatewayBasePathMappingRead,
+		Delete: resourceAwsApiGatewayBasePathMappingDelete,
```

Comment:
It seems to be possible to update certain fields of this resource:
http://docs.aws.amazon.com/cli/latest/reference/apigateway/update-base-path-mapping.html

but I'm totally happy to treat this PR as 1st iteration and eventually add `Update()` in a followup PR.